packages/core/src/__test__/wallet.mock.ts

```diff
@@ -1,6 +1,5 @@
-import { StarknetWindowObject } from "../StarknetWindowObject"
 import wallets from "../discovery"
-import { Permission } from "../rpcMessage"
+import type { Permission, StarknetWindowObject } from "starknet-types"
```

Comment:
Please import Permission without the type alias

Reply:
Done